Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# 27th Bug Crusade

## Changelog:
- Fixed #1183 issue where `upgrade_condemned_halls_desc`, an upgrade of the **Twilight Citadel** wonder, had no localization.
**_How to test:_** just a tip, there's a Twilight Citadel in Ahn'qiraj.
- Fixed #1199 issue where Northridge province was tribal.
- **Ruled too long** opinion should no longer appear in the game. Fixes #809.
**_How to test:_** wait and watch!
- Fixed #1275 issue where the event, that appears when you try to marry someone on an incompatible race, missed spouse names in localization.
- Fixed #822 issue where Old Gods' cultists were glad when C'Thun dies in `WCOME.201` event.
**_How to test:_** use `event WCOME.201` being a cultist/non-cultist.
- Fixed #1278 issue where the Plague of Undeath stopped developing (into an undead) after the first symptom.
- Fixed #1277 issue where the Plague diagnosis text missed a word and didn't mention that you got the Plague.


## Developer changelog:
- Fixed #1223 issue where **Cenarion Enclave** building was visible in all the temples and hospitals.
- **Cenarion Enclave** building is no longer active if you or your liege follows Dark religion (like Hakkari, Helya, etc.) Fixes #1224.
- All the racial divorce texts talk about the same things in the same tense.
- `EVTDESC_WCCLN_113` talks about "both parties" instead of "you".

## Scripting changelog (ignore if you can't read CK2 script):
- Fixed issue where diseases increased the class level up's speed instead of decreasing it.
test:
